runtime/syntax/pamenv.vim

```diff
@@ -0,0 +1,29 @@
+" Vim syntax file
+" Language:             pam_env.conf(5) configuration file
+" Latest Revision:      2020-05-10
+
+if exists("b:current_syntax")
+  finish
+endif
+
+syn keyword     pamenvTodo              contained TODO FIXME XXX NOTE
+syn region      pamenvComment           start='^#' end='$' display oneline contains=pamenvTodo,@Spells
+
+syn match       pamenvVars              '^[A-Z_][A-Z_0-9]*' nextgroup=pamenvKeywords skipwhite
+
+syn keyword     pamenvKeywords          contained DEFAULT OVERRIDE nextgroup=pamenvVarEq
+
+syn match       pamenvVarEq             contained '='
+                                        \ nextgroup=pamenvValue,pamenvValueWithQuote
+
+syn match       pamenvValue             contained '[^ \t]*' skipwhite nextgroup=pamenvKeywords
+syn region      pamenvValueWithQuote    contained start='"' end='"' skipwhite nextgroup=pamenvKeywords
```

Comment:
It would probably be nice to recognize and highlight backslash escapes and variable substitutions.

https://manpages.debian.org/buster/libpam-modules/pam_env.conf.5.en.html has examples:

```
      DOLLAR         DEFAULT=\$
      DOLLARDOLLAR   DEFAULT=        OVERRIDE=\$${DOLLAR}
      DOLLARPLUS     DEFAULT=\${REMOTEHOST}${REMOTEHOST}
      ATSIGN         DEFAULT=""      OVERRIDE=\@
```

Note that variable substitutions use either `$` or `@` (one expands envvars, the other passwd fields):

```
      PATH           DEFAULT=${HOME}/bin:/usr/local/bin:/bin\
      :/usr/bin:/usr/local/bin/X11:/usr/bin/X11
      XDG_DATA_HOME  @{HOME}/share/
```

Reply:
I want that too. But my vim script skill is not good. Maybe leave an TODO for others.